src/app/pages/administration/administration.state.js

```diff
@@ -85,6 +85,9 @@ const states = [{
     roles: ['chpl-admin', 'chpl-onc', 'chpl-onc-acb'],
   },
 }, {
+  name: 'administration-redirect',
+  url:'/#/',
```

Comment:
There is no need to create a new route. In your `state.go` command, you can just route them to `'search'`